servicex/servicex_client.py

```diff
@@ -82,40 +117,61 @@ def get_codegen(_sample: Sample, _general: General):
                 if sample.Tree:
                     query = query.set_tree(sample.Tree)
                 sample.Query = query
-
+                logger.debug(f"final qastle_query: {sample.Query.generate_selection_string()}")
+            elif isinstance(sample.Query, PythonQuery):
```

Comment:
What is the motivation to add this branch? (I thought the interpretation of PythonQuery worked fine already)